heimdall-middleware-spec/src/test/java/br/com/conductor/heimdall/middleware/util/helpermock/call/RequestMockTest.java

```diff
@@ -0,0 +1,50 @@
+package br.com.conductor.heimdall.middleware.util.helpermock.call;
+
+import org.junit.Test;
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
```

Comment:
I think this static import is probably ok. I've just imported each Assert method separately in a recent PR to save importing unnecessary stuff but, looking at other tests in this project, I can see examples of both as well another case where org.junit/Assert is imported and each assert is prefixed "Assert."